### PR TITLE
Update sign-in screen to latest mocks

### LINKED ISSUE #1866: Update sign in screen to match latest mocks
- [ ] Display background (already in resources)
- [ ] Add gradient overflay
- [ ] Tweak size, font, and position of logo and title

image

### PR DESCRIPTION
**Pull request by gino-m**
Implemented using a responsible ConstraintLayout.

Closes #1866

Portrait:

Landscape:

### CODE REVIEW

File: ground/src/main/res/layout/sign_in_frag.xml [hunk 1]
```diff
@@ -25,62 +25,71 @@
...
       android:layout_width="match_parent"
-      android:layout_height="match_parent">
+      android:layout_height="match_parent"
+      android:alpha="0.40"
+      android:background="#146C2E"
+      app:layout_constraintHeight="match_parent"
+      app:layout_constraintWidth="match_parent" />
+
+    <ImageView
+      android:layout_width="528dp"
```
Comment (shobhitagarwal1612) [inline ground/src/main/res/layout/sign_in_frag.xml:50]: Let's keep the width as `match_parent`

File: ground/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -62,6 +62,13 @@
     <item name="colorPrimaryInverse">@color/md_theme_inversePrimary</item>
   </style>
 
+  <style name="AppTitle">
+    <item name="android:textColor">?attr/colorOnPrimary</item>
```
Comment (shobhitagarwal1612) [inline ground/src/main/res/values/styles.xml:66]: We should use `@android:color/white` as the text color here.

**Review by shobhitagarwal1612 — CHANGES_REQUESTED** [review]

File: ground/src/main/res/layout/sign_in_frag.xml (hunk 1 above)
Comment (gino-m) [reply ground/src/main/res/layout/sign_in_frag.xml:50]: Done

File: ground/src/main/res/values/styles.xml (hunk 2 above)
Comment (gino-m) [reply ground/src/main/res/values/styles.xml:66]: Done

**Comment by gino-m** [comment]
Applied suggestions and new font from @vittorino and updated above screenshots.

@shobhitagarwal1612 PTAL?

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**